lib/bloc/podcast_payments/podcast_payments_bloc.dart

```diff
@@ -110,6 +112,17 @@ class PodcastPaymentsBloc with AsyncActionsHandler {
               seconds: _listeningTime[currentPlayedEpisode.contentUrl].floor())
           .inMinutes;
 
+      if (_listeningTime[currentPlayedEpisode.contentUrl].floor() %
+              updatePodcastHistoryFrequencyInSeconds ==
+          0) {
+        _addToPodcastHistory(
```

Comment:
shouldn't we add to the podcast history after we pay?

Reply:
@roeierez It's also recorded. Please see https://github.com/breez/breezmobile/blob/ab6ba1213bd60bfa12a86ddc544eacd7a2c10bba/lib/bloc/podcast_payments/podcast_payments_bloc.dart#L266